### ISSUE TITLE
Feature Request: Allow external management of controller

### ISSUE THREAD

**Issue by maltegrosse** [zyclonite/zerotier-docker#8]
Hey, thank you for your great container.

as far as I understood it correctly, your container could be used as client or controller container.
Thus, introducing an environment variable (e.g. `ALLOW_EXTERNAL`) would be helpful to have external access to the controller.

An additional entrypoint.sh for the non bridge container could be created which includes something like
```
if [[ ! -z "${ALLOW_EXTERNAL}" ]]; then
  echo "{\"settings\": {\"portMappingEnabled\": true,\"softwareUpdate\": \"disable\",\"allowManagementFrom\": [\"0.0.0.0/0\"]}}" > /var/lib/zerotier-one/local.conf
fi
```

**Comment by dec0dOS** [comment]
Hello, @zyclonite. Thanks for your work, it's a valuable contribution.
I'm the maintainer https://github.com/dec0dOS/zero-ui. We're using `zyclonite/zerotier-docker` as a [base image for ZeroTier controller image](https://github.com/dec0dOS/zero-ui/blob/main/docker/zerotier/Dockerfile). Adding the lines above to the `entrypoint.sh` could help get rid of rebuilding the image with the custom `local.conf` and directly including to the [docker-compose.yml example](https://github.com/dec0dOS/zero-ui/blob/main/docker-compose.yml).

Github Actions sometimes fail with strange errors in our case (https://github.com/dec0dOS/zero-ui/runs/4575459460?check_suite_focus=true), so rebuilding the image is a waste of time and resources.

Thanks!

**Comment by zyclonite** [comment]
hi, i gave it a try in this commit https://github.com/zyclonite/zerotier-docker/commit/52ee605ed16f424ceaa80143b070431e6a81d8af

i have currently some issues with the multiarch builds because of a buildah bug
as long as you are using amd64, the latest main image should include the feature

let me know if that would work

**Comment by dec0dOS** [comment]
@zyclonite, many thanks!
Seems to be the `zyclonite/zerotier:main` image is working as expected!

**Comment by zyclonite** [comment]
@dec0dOS released version 1.10.0 with this feature now

**Comment by maltegrosse** [comment]
thank you for implementing this feature
